api/workerapi/v1/worker.proto

```diff
@@ -0,0 +1,78 @@
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+import "patch/go.proto";
+
+option (go.lint).all = true;
+option (go.lint).initialisms = "ID";
+
+// Versioning via package name, for future-proofing
+package wharf.worker.v1;
+
+option go_package = "github.com/iver-wharf/wharf-cmd/api/workerapi/v1";
+
+service Worker {
+  rpc StreamLogs(LogStreamRequest) returns (stream LogStream);
+  rpc StreamStatusEvents(StatusEventRequest) returns (stream StatusEvent);
+  rpc StreamArtifactEvents(ArtifactEventRequest) returns (stream ArtifactEvent);
+}
+
+message LogStreamRequest {
+  int32 chunkSize = 1;
+}
+// Empty messages, but exists for potential future usages
+message StatusEventRequest { }
+message ArtifactEventRequest { }
+
+message LogStream {
+  // Lines is an array of log lines to allow chunking log line streams.
+  repeated LogLine lines = 1;
+}
```

Comment:
On second look, I've noticed that we don't need the `repeated` functionality. It gives us zero performance boost as gRPC already buffers for us. https://github.com/iver-wharf/wharf-cmd/issues/14#issuecomment-1056635236

We should change this back to only including a single LogLine. Perhaps even just embedding the `LogLine` type inside `LogStream`